Cargo.lock

```diff
@@ -1534,6 +1652,7 @@ dependencies = [
  "libp2p-dns",
  "libp2p-gossipsub",
  "libp2p-mplex",
+ "libp2p-noise",
  "libp2p-request-response",
  "libp2p-secio",
```

Comment:
Why is this dependency not going away? :thinking:

Reply:
Sloppy jalopy - will fix. Thanks.

Reply:
oh not so sloppy, I did remove the feature from libp2p.

Reply:
Found the little blighter - it was in comit network tests.

Reply:
`git grep -i secio` now returns nothing :)